### DISCUSSION TITLE
关于 IoTDB 的几个问题

### DISCUSSION THREAD

**Discussion by StrongManLi** [apache/iotdb#11449] [Q&A] [answered]
1. 内网环境下安装了 IoTDB，本地启动成功，启动客户端成功，执行 show timeseries 成功；
Shell 执行 curl ip:6667 Empty Reply（这正常吗），其他机器 curl 不通没反应，Java项目访问不通（其他机器）；
请问 IoTDB 有关于远程访问的相关配置吗？端口什么的我这边都是通的。
2. /etc/hosts 文件中没配置 hostname（hostname命令的返回值），这个时候经常启动失败，而且启动巨慢（在我的服务器上）。这个也有可能是缺少某些命令导致的？有lsof。
3.

**Comment by CritasWang** [comment, marked as answer]
1. iotdb 的 6667 端口 不是 http是 rpc ，所以无法通过 curl 查看
远程访问的话 可以通过 start-cli.sh -h ip 来连接，如果无法连接，那在iotdb 所在机器上用  start-cli.sh -h ip  来连接，如果能正常连接，那代表 iotdb 已经在对应的 ip+port 启动了，如果无法连接，则需要确认 iotdb-datanode.properties 文件 中 dn_rpc_address 地址配置的是什么。
2. /etc/hosts 文件中没配置 hostname，本机的 hostname 建议还是配置上
